pkg/interop/messages.go

```diff
@@ -58,6 +58,18 @@ type ErrorMessage struct {
 	Result Result
 }
 
+// NsMessageHeader contains the message header for NS messages.
+type NsMessageHeader struct {
+	MessageHeader
+	SenderID   NetID
```

Comment:
Is the `0x` handling supposed to be removed ? I expected it to be in the unmarshaller maybe.

Reply:
Yeah it's there, it goes via `Buffer`: https://github.com/TheThingsNetwork/lorawan-stack/blob/feature/4677-interop/pkg/interop/types.go#L279-L287